Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -4417,26 +4417,21 @@ private void HighlightElement(ElementInfo ElementInfo, bool locateElementByItLoc
                 SwitchFrame(ElementInfo.Path, ElementInfo.XPath, true);
 
                 //Find element 
-
-                if ((IWebElement)ElementInfo.ElementObject == null)
+                if (locateElementByItLocators)
```

Comment:
this is fine but why you removed the Object cleanup on POM page when Agent is been changed? please add it back on top of this change.